sql/sqlite/driver.go

```diff
@@ -60,6 +60,16 @@ func init() {
 			return uc
 		})),
 	)
+	sqlclient.Register(
+		"libsql",
+		sqlclient.DriverOpener(Open),
+		sqlclient.RegisterTxOpener(OpenTx),
+		sqlclient.RegisterCodec(MarshalHCL, EvalHCL),
+		sqlclient.RegisterFlavours("libsql+wss", "libsql+https"),
```

Comment:
At the moment +https won't work as it does not support interactive transactions and is missing some features. We're working on making it better but maybe it's worth to keep https out of scope for this PR?